roles-in-tech/security.md

```diff
@@ -12,7 +12,7 @@
 - [Intro to Security](/security/web-security.md)
 
 ### Motivation
-- Computer security is important, primarily to *keep your information protected*. It’s also important for your computer’s overall health, helping to prevent viruses and malware and allowing programs to run more smoothly.
+- Computer security is important, primarily to *keep your information protected*. It’s also important for your computer’s overall health, helping to prevent viruses and malware and allowing programs to run more smoothly. Refer more at [The Importance of Computer Security](https://blog.onsharp.com/the-importance-of-computer-security)
```

Comment:
Refer more at [The Importance of Computer Security](https://blog.onsharp.com/the-importance-of-computer-security)

Change to just:
([The Importance of Computer Security](https://blog.onsharp.com/the-importance-of-computer-security))